olclient/cli.py

```diff
@@ -88,7 +88,18 @@ def main():
         config_tool.update(config)
         return "Successfully configured "
 
-    ol = OpenLibrary()
+    # prompt first time users to configure their OpenLibrary credentials
```

Comment:
I think this should be / is on a different branch and shouldn't be added to this PR.

You can use:
`git add --patch -i` to specify which parts of your working branch you want committed.

Reply:
My bad @mekarpeles,
I have raised another PR with only relevant commits.